Ashasindhu.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Asha Gadiraju",
+  "displayStatsPublic":true,
```

Comment:
`displayStatsPublic` is not required. Linking you to an example JSON doc https://linkfree.eddiehub.io/docs/full-profile-example